it/mail/chrome/messenger/localMsgs.properties

```diff
@@ -230,39 +230,39 @@ pop3PreviouslyEnteredPasswordIsInvalidPrompt=Please enter a new password for use
+4047=Errore di stato interno durante l'autenticazione al server POP3. Questo è un errore interno e imprevisto nell'applicazione, per favore segnalalo come bug.
 
 ## @name POP3_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 ## @loc None
-4048=This POP3 server does not seem to support encrypted passwords. If you just set up the account, please try changing to 'Password, transmitted insecurely' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, this is a common scenario how someone could steal your password.
+4048=Sembra che questo server POP3 non supoorti le password crittografate. Se hai appena impostato l'account, prova a selezionare 'Password, transmessa in maniera non sicura' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'. Se prima funzionava e ora improvvisamente non funziona, questo è uno scenario comune come se qualcuno potesse rubare la tua password.
 
 ## @name POP3_AUTH_CHANGE_ENCRYPT_TO_PLAIN_SSL
 ## @loc None
-4049=This POP3 server does not seem to support encrypted passwords. If you just set up this account, please try changing to 'Normal password' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, please contact your email administrator or provider.
+4049=Sembra che questo server POP3 non supoorti le password crittografate. Se hai appena impostato l'account, prova a selezionare 'Password normale' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'. Se prima funzionava e ora improvvisamente non funziona, per favore contatta il tuo amministrtore o fornitore di servizi e-mail.
 
 ## @name POP3_AUTH_CHANGE_PLAIN_TO_ENCRYPT
 ## @loc None
-4050=This POP3 server does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+4050=Questo server POP3 non consente password con testo semplice. Prova a selezionare 'Password crittografate' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'.
 
 # Authentication server caps and pref don't match
 ## @name POP3_AUTH_MECH_NOT_SUPPORTED
 ## @loc None
-4051=The server does not support the selected authentication method. Please change the 'Authentication method' in the 'Account Settings | Server settings'.
+4051=Il server non supporta il metodo di autenticazione selezionato. Cambia il 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'.
 
 # Status - Could not log in to GSSAPI, and it was the only method
 ## @name POP3_GSSAPI_FAILURE
 ## @loc None
-4052=The Kerberos/GSSAPI ticket was not accepted by the POP server. Please check that you are logged in to the Kerberos/GSSAPI realm.
+4052=Il ticket Kerberos/GSSAPI non è stato accettato dal server POP. Controlla di avere effettuato l'accesso nel reame Kerberos/GSSAPI.
```

Comment:
I don't think that 'dominio' (domain) is the right translation. I think it is a protocol. I would translate realm literally as 'reame' e.g. official documentation Ubuntu: http://help.ubuntu-it.org/10.10/ubuntu/serverguide/it/kerberos.html

4052=Il ticket Kerberos/GSSAPI non è stato accettato dal server POP. Controlla di avere effettuato l'accesso nel reame Kerberos/GSSAPI.